moviepy/config.py

```diff
@@ -56,7 +56,11 @@ def try_cmd(cmd):
             IMAGEMAGICK_BINARY = wr.QueryValueEx(key, 'BinPath')[0] + r"\convert.exe"
             key.Close()
         except:
-            IMAGEMAGICK_BINARY = 'unset'
+            try:
+                IMAGEMAGICK_PATH   = sp.check_output('dir /B /O-N "C:\Program Files\ImageMagick-*"'                           , shell=True).decode("utf-8").split("\r\n")[0]
+                IMAGEMAGICK_BINARY = sp.check_output('dir /B /S   "C:\Program Files\{}\*convert.exe"'.format(IMAGEMAGICK_PATH), shell=True).decode("utf-8").split("\r\n")[0]
```

Comment:
This looks like it may be pretty useful. The only thing I'd comment is that you've put `IMAGEMAGICK_PATH` in all-caps which by convention means that it is a constant used by the program in a variety of places. Here, though, it is just a variable to increase readability so I'd suggest renaming it to `imagemagick_path` or something like that.

Reply:
sure it's done